pkg/repository/git.go

```diff
@@ -100,18 +104,18 @@ func (r gitRepoController) isClean(ctx context.Context) (bool, error) {
 func (r gitRepoController) getSHA() (string, error) {
 	isClean, err := r.isClean(context.TODO())
```

Comment:
```suggestion
	isClean, err := r.isClean(context.Background())
```

should this not be TODO ?

Reply:
afaik `TODO` is theorically correct because we don't have the context linked to another one and we should only have one `context.Background` but I need to research more about this to give a better insight